src/traits.rs

```diff
@@ -31,16 +31,17 @@ pub trait Instantiate {
     ///
     /// ## Example
     /// ```
-    /// #[cfg_attr(not(feature = "library"), entry_point)]
-    /// pub fn instantiate(
-    ///     deps: DepsMut,
-    ///     env: Env,
-    ///     info: MessageInfo,
-    ///     msg: InstantiateMsg,
-    /// ) -> Result<Response, ContractError> {
-    ///     let my_token = MyToken::new(...);
-    ///     my_token.instantiate(deps, to_binary(&msg.init_info)?)
-    /// }
+    /// // TODO: fix this example
```

Comment:
Just use the `ignore` attribute, since this example is not meant to compile. See https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes

Reply:
fixed